tests/Feature/MembershipTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Hearth\Tests\Feature;
+
+use App\Models\Organization;
+use App\Models\User;
+use Hearth\Models\Membership;
+use Hearth\Tests\TestCase;
+use Illuminate\Support\Facades\Schema;
+
+class MembershipTest extends TestCase
+{
+    protected function getEnvironmentSetUp($app)
+    {
+        Schema::dropAllTables();
+
+        $create_users_table = include __DIR__ . '/../../database/migrations/create_users_table.php.stub';
+        $update_users_table = include __DIR__ . '/../../database/migrations/update_users_table.php.stub';
+        $create_memberships_table = include __DIR__ . '/../../database/migrations/2021_03_01_000000_create_memberships_table.php';
+        $create_organizations_table = include __DIR__ . '/../../database/migrations/create_organizations_table.php.stub';
+
+        $create_users_table->up();
+        $update_users_table->up();
+        $create_memberships_table->up();
+        $create_organizations_table->up();
+    }
```

Comment:
Not sure if this will work for you, but I used the `DatabaseMigrations` trait to run the migrations with the tests.